TISServices/TISServices/TISServices.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 15

```

Comment:
This file was modified as a result of using a newer version of Visual Studio and adding the new unit test project to the solution